packages/manager/modules/pci/src/projects/project/quota/quota.html

```diff
@@ -0,0 +1,92 @@
+<pci-project-quota-header></pci-project-quota-header>
+
+<cui-message-container data-messages="$ctrl.messages"></cui-message-container>
+
+<div class="text-center" data-ng-if="$ctrl.isLoading">
+     <oui-spinner></oui-spinner>
+</div>
+
+<div class="pt-2" data-ng-if="!$ctrl.isLoading">
```

Comment:
Could be a `mb-2`  class to the previous node instead?

In order to keep consistency and apply  base margin always in the same direction.